.cursor/rules/jira-ticket.mdc

```diff
@@ -143,26 +190,87 @@ h2. Additional context
 
 ## Severity classification
 
-- SEV1: Critical - System down, data loss, security breach
-- SEV2: High - Major feature broken, significant user impact
-- SEV3: Medium - Feature partially broken, moderate impact
-- SEV4: Low - Minor issues, minimal user impact
-- SEV5: Trivial - Cosmetic issues, no functional impact
+Use this guide to determine the appropriate severity level:
+
+**SEV1 (Critical)** - Immediate action required:
+
+- Production system is down
+- Data loss or corruption
+- Security vulnerability or breach
+- Complete feature failure affecting all users
+
+**SEV2 (High)** - Action required within 24-48 hours:
+
+- Major feature is broken for most users
+- Significant performance degradation
+- Workaround exists but is impractical
+- Blocking progress on dependent work
+
+**SEV3 (Medium)** - Action required within 1-2 weeks:
+
+- Feature partially broken or unreliable
+- Affects subset of users
+- Reasonable workaround available
+- Non-critical functionality impaired
+
+**SEV4 (Low)** - Action required within a month:
+
+- Minor bugs with minimal impact
+- Edge case issues
+- Nice-to-have improvements
+- Non-blocking technical debt
+
+**SEV5 (Trivial)** - No urgency:
+
+- Cosmetic issues only
+- Documentation typos
+- Code formatting
+- Optional optimizations
```

Comment:
Can we add bullets about accessibility issues for each of the severity levels?

Reply:
would you mind suggesting the language you would want to use for each level?